src/renderer/api/kube-watch-api.ts

```diff
@@ -51,6 +52,12 @@ export class KubeWatchApi {
   @observable subscribers = observable.map<KubeApi, number>();
   @observable isConnected = false;
 
+  @computed get watchAllEnabled(): boolean {
+    const { isAdmin, watchApiAllEnabled } = this?.cluster;
```

Comment:
I think we don't need `isAdmin` check here.

Reply:
Fixed.